sympy/tensor/toperators.py

```diff
@@ -22,12 +23,15 @@ class PartialDerivative(TensExpr):
     The ``PartialDerivative`` object behaves like a tensorial expression:
 
     >>> expr.get_indices()
-    [i, j]
+    [i, -j]
 
     Indices can be contracted:
 
-    >>> PartialDerivative(A(i), A(-i))
-    PartialDerivative(A(L_0), A(-L_0))
+    >>> expr = PartialDerivative(A(i), A(i))
+    >>> expr
+    PartialDerivative(A(L_0), A(L_0))
+    >>> expr.get_indices()
+    [L_0, -L_0]
```

Comment:
@joha2 I have fixed it like this. `PartialDerivative(A(i), A(i))` gets printed fully-contravariant. When you access the indices, one is contravariant, while the deriving index gets flipped into covariant.

Indeed, deriving by a contravariant symbol adds a covariant slot to the resulting tensor.

Reply:
I think as long as the pretty print is okay, people can live with the non intuitive output of a non-performed `PartialDerivative` :-)

Reply:
I have also pushed the LaTeX printers, so people should be able to see a nice output in Jupyter notebooks.